pom.xml

```diff
@@ -1060,6 +1081,64 @@
             </modules>
         </profile>
 
+        <profile>
+            <id>jdk-8</id>
+            <activation>
+                <jdk>[1.8,)</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <version>${maven.compiler.plugin.version}</version>
+                        <configuration>
+                            <source>1.8</source>
+                            <target>1.8</target>
+                            <encoding>${project.build.sourceEncoding}</encoding>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <!--
+            Profile for execution of test suite only on JDKs prior to 1.8.
+            Not suitable for building Hazelcast.
+            Sample usage:
+            1. Build JDK 6 artifacts using JDK8:
+               $ mvn -P-jdk-8 -DskipTests -T1C clean install
```

Comment:
Maybe it is worth having an extra comment to highlight the hyphen (and its purpose) here, to deactivate the `jdk-8` profile.

Reply:
done, it is easy to miss the profile deactivation "-"